<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogo_BusquedaDatos</class>
 <widget class="QDialog" name="Dialogo_BusquedaDatos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SQLite Busqueda Informacion</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_id">
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txt_id"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="lbl_nombre">
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="txt_nombre">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="btn_buscar">
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
